app/components/answer-step.jsx

```diff
@@ -0,0 +1,117 @@
+// https://github.com/EnCiv/civil-pursuit/issues/102
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import StepIntro from './step-intro'
+import WhyInput from './why-input'
+import ObjectId from 'bson-objectid'
+
+const AnswerStep = props => {
+  const {
+    className = '',
+    intro = '', // text for the introduction
+    whyQuestion1 = '', // the text for the left of the first why input
```

Comment:
question is for the upper why input.
whyQuestion is for the lower why input.  Since it's not a point object you will need to do `<WhyInput point={{description: whyQuestion}} ... /> to make it work.